src/Sylius/Behat/Context/Ui/Admin/RemovingTaxonContext.php

```diff
@@ -32,5 +38,28 @@ public function iRemoveTaxonNamed(string $name): void
     {
         $this->createPage->open();
         $this->createPage->deleteTaxonOnPageByName($name);
+
+        $this->sharedStorage->set('taxon_name', $name);
+    }
+
+    /**
+     * @Then the taxon should still exist
```

Comment:
```suggestion
     * @Then /^(this taxon) should still exist$/
```
Then you could remove all the taxon name hijinks